dotcom-rendering/cypress/integration/parallel-3/article.interactivity.spec.js

```diff
@@ -36,27 +36,74 @@ describe('Interactivity', function () {
 			cy.visit(`/Article?url=${articleUrl}`);
 			cy.get('[data-cy=share-counts]').should('exist');
 		});
-		it('should display all the rich links for an article', function () {
+		it('loads the discussion when you click the comment count', function () {
+			cy.visit(
+				`/Article?url=https://www.theguardian.com/commentisfree/2022/jan/20/uk-government-yemen-war-saudi-arabia-westminster`,
+			);
+			cy.get('[data-cy=comment-counts]').should('exist');
+			// The discusion is not yet loaded
+			cy.get('[data-component=discussion]').should('not.exist');
+			// Click the comment count
+			cy.get('[data-cy=comment-counts]').click();
+			cy.get('[data-component=discussion]').should('exist');
+		});
+		it('loads the discussion immediately when you use a url ending in #comments', function () {
+			cy.visit(
+				`/Article?url=https://www.theguardian.com/commentisfree/2022/jan/20/uk-government-yemen-war-saudi-arabia-westminster#comments`,
+			);
+			cy.get('[data-component=discussion]').should('exist');
```

Comment:
Just trying to wrap my head around the selection. `[data-component=discussion]` is a child of `gu-island[name=DiscussionContainer]`, and also only loaded in async?

Reply:
When do we use `data-component`?

Reply:
So we use `data-component` for Ophan. We shouldn't be using that here as it's coupling the tests with the tracking solution, I'll update this